doc/source/Platforms.rst

```diff
@@ -548,9 +560,9 @@ The following instructions were used to prepare a basic Ubuntu 20.04 system as i
    apt-get upgrade
 
    # Compilers
+   apt install -y gcc g++ gfortran gdb
```

Comment:
This allows us to use either gcc-9.4.0 or gcc-10.3.0.

Reply:
I'm not sure what you mean? As in depending on which version of (i.e. Ubuntu 20, 18) you're using it'll install the default?